src/cogniteClient.ts

```diff
@@ -291,12 +291,64 @@ export default class CogniteClient {
   public getMetadata = (value: any) => this.metadataMap.get(value);
 
   /**
-   * Do plain get-requests using the client
+   * Basic HTTP method for GET
+   *
+   * @param path The URL path
+   * @param options Request options, optional
+   *
+   * ```js
+   * import { CogniteClient } from '@cognite/sdk';
+   * const client = new CogniteClient({ appId: '[APP NAME]' });
+   * client.loginWithApiKey({
+   *  project: '[PROJECT]',
+   *  apiKey: '[API_KEY]'
+   * });
+   *
+   * // Get all assets for this client
+   * const assets = await client.get('/api/v1/projects/{project}/assets');
```

Comment:
Great. I would have called it `const response` since you don't get the assets directly(`{items: []}`)